src/handlers/beta_backport.rs

```diff
@@ -0,0 +1,52 @@
+use crate::github::{Event, IssuesAction, Label};
+use crate::handlers::Context;
+use regex::Regex;
+
+lazy_static! {
+    // See https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/linking-a-pull-request-to-an-issue
+    // Max 19 digits long to prevent u64 overflow
+    static ref CLOSES_ISSUE: Regex = Regex::new("(close[sd]|fix(e[sd])?|resolve[sd]) #(\\d{1,19})").unwrap();
+}
+
+pub(crate) async fn handle(
+    ctx: &Context,
+    event: &Event,
+) -> anyhow::Result<()> {
+    let issue_event = if let Event::Issue(event) = event {
+        event
+    } else {
+        return Ok(());
+    };
+
+    if issue_event.action != IssuesAction::Opened {
+        return Ok(());
+    }
+
+    if issue_event.issue.pull_request.is_none() {
+        return Ok(());
+    }
+
+    for caps in CLOSES_ISSUE.captures_iter(&issue_event.issue.body) {
+        // Should never fail due to the regex
+        let issue_id = caps.get(1).unwrap().as_str().parse::<u64>().unwrap();
```

Comment:
I would prefer to handle failure to parse here, likely by early returning, rather than with a digit count in the regex.